api/valor_api/backend/metrics/classification.py

```diff
@@ -981,26 +1154,173 @@ def _compute_confusion_matrix_and_metrics_at_grouper_key(
             ),
         ]
 
-    return confusion_matrix, output
+    return [
+        confusion_matrix
+        for confusion_matrix in confusion_matrices.values()
+        if confusion_matrix
+    ], metrics
 
 
-def _compute_clf_metrics(
+def _aggregate_data(
```

Comment:
this name isn't descriptive; could we change it back to something to do with computing classification metrics (per the docstring)?

Reply:
The function's role is to collect data, It has nothing to do with computing metrics so im a little confused.

Ill update the docstring header since that is wrong!